cob_hardware_config/cob4-3/config/base_controller.yaml

```diff
@@ -1,5 +1,5 @@
 ## joint_names
-joint_names: [fl_caster_rotation_joint, fl_caster_r_wheel_joint, fr_caster_rotation_joint, fr_caster_r_wheel_joint, b_caster_rotation_joint, b_caster_r_wheel_joint]
+joint_names: [fl_caster_rotation_joint, fl_caster_r_wheel_joint, b_caster_rotation_joint, b_caster_r_wheel_joint, fr_caster_rotation_joint, fr_caster_r_wheel_joint]
```

Comment:
The `joint_names` parameter is not used by the driver or the controllers.. I believe it is used by `cob_script_server`/`cob_command_gui`